### ISSUE TITLE
Vulnerabilities from jquery doc scripts

### ISSUE THREAD

**Issue by tomsoal** [json-editor/json-editor#1070]
### General information

- json-editor version: latest (2.6.0)

### Expected behavior
Should not raise any vulnerabilities

### Actual behavior
Multiple vulnerabilities raised for script usage of jquery plugin
e.g. https://github.com/json-editor/json-editor/blob/master/docs/select2.html#L7, https://github.com/json-editor/json-editor/blob/master/docs/materialize_css.html#L78

Causes the following vulnerability - https://vuln.whitesourcesoftware.com/vulnerability/CVE-2019-11358

### Steps to reproduce the behavior

Direct link to example: https://json-editor.github.io/json-editor/

```javascript
{
  "title": "json schema example",
  "type": "object",
  "properties": {
    "example": {
      "type": "string",
      "description": "This is an example schema.",
      "default": "Please edit me."
    }
  }
}
```

**Comment by schmunk42** [comment]
Isn't this fixed in 3.4.0?
The demo here https://github.com/json-editor/json-editor/blob/master/docs/select2.html#L7 includes 3.4.1

**Comment by tomsoal** [comment]
hmm good spot, also seen https://www.whitesourcesoftware.com/vulnerability-database/CVE-2020-11023 which requires 3.5.0 so assume its best to jump to the latest version

Have made the potential change so will raise a PR to change these scripts to the latest version

**Comment by tomsoal** [comment]
PR has been merged now so vulnerability addressed, thank you!

**Comment by tomsoal** [comment]
Hi @schmunk42 should've thought of this before - any chance a new tag could be made for this change please? Would be helpful for future reference to know which subversion is clear of the vulnerability 😄

**Comment by schmunk42** [comment]
Available in 2.6.1 ;)
